README.md

```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
 - BlackBerry 10
 - Windows Phone 7 and 8
 - Windows (Windows Phone 8.1 only)
-- Tizen
 - Firefox OS
-- Browser
+- Browsers
```

Comment:
Browser is the name of the platform. For the supported ones, maybe just list them in parentheses like windows does above